apiml/src/main/resources/application.yml

```
@@ -44,9 +44,17 @@ spring:
 
 springdoc:
     api-docs:
-        path: /gateway/api-docs
+        path: /v3/api-docs
+        groups:
+            enabled: true
     writer-with-order-by-keys: true
-    pathsToMatch: /gateway/api/v1/**
+    group-configs:
+        - group: gateway
+          pathsToMatch: /gateway/api/v1/**
+        - group: cachingservice
+          paths-to-match: /cachingservice/api/v1/**
```

```suggestion
        - group: gateway
          pathsToMatch: /gateway/api/v1/**
        - group: cachingservice
          paths-to-match: /cachingservice/api/v1/**
        - group: discovery
          paths-to-match: /discovery/api/v1/**
        - group: zaas
          paths-to-match: /zaas/api/v1/**
```

Discovery service is missing (see `/discovery/api/v1/staticApi`), and I guess we should define ZAAS to be completed as well.